app/utils/callback/tests/data/lava-json-meson-gxbb-p200.json

```diff
@@ -0,0 +1 @@
958c82dfe44e28ed142f21a2310c7dc47616b603201a963d78c674f}\n    - {sha512sum: 8475411cc73fdb6d651d5c2a21297fe7300d99e0054d8c46602af7748c1cafe574b7bad7bb90102b4a0168f98a5a071d3d006fae834d02f611621e3798f7f1f1}\n    - {size: 30403072}\n    level: 1.2.1\n    namespace: common\n    result: pass\n  name: http-download\n  result: pass\n  suite: lava\n  unit: seconds\n  url: /results/testcase/7782477\n- id: '7782473'\n  job: '370874'\n  level: 1.1.1\n  log_end_line: ''\n  log_start_line: ''\n  logged: '2020-11-19 07:03:30.302230+00:00'\n  measurement: '0.6400000000'\n  metadata:\n    case: http-download\n    definition: lava\n    duration: '0.64'\n    extra:\n    - {label: ramdisk}\n    - {md5sum: 86d8d8f5b58abaf7cdddbd5e53df7efc}\n    - {sha256sum: 46e561b3cade517c2eac14b796a9b857151a4ddf72246454393d18d86d99ef8e}\n    - {sha512sum: 3551d3ddb9b6dc1df7016339af072f61a03540f24349ecca33c98184dda6ac59fc3d8482a244328385b1f1190cf21e2bdca4ff0920a08e966eb730c373b58bdf}\n    - {size: 4984326}\n    level: 1.1.1\n    namespace: common\n    result: pass\n  name: http-download\n  result: pass\n  suite: lava\n  unit: seconds\n  url: /results/testcase/7782473\n- id: '7782471'\n  job: '370874'\n  level: ''\n  log_end_line: ''\n  log_start_line: ''\n  logged: '2020-11-19 07:03:30.302162+00:00'\n  measurement: None\n  metadata: {case: validate, definition: lava, result: pass}\n  name: validate\n  result: pass\n  suite: lava\n  unit: ''\n  url: /results/testcase/7782471\n", "0_dmesg": "- id: '7782929'\n  job: '370874'\n  level: ''\n  log_end_line: '930'\n  log_start_line: '930'\n  logged: '2020-11-19 07:06:12.073984+00:00'\n  measurement: '2.0000000000'\n  metadata: {case: emerg, definition: 0_dmesg, measurement: '2', result: fail, units: lines}\n  name: emerg\n  result: fail\n  suite: 0_dmesg\n  unit: lines\n  url: /results/testcase/7782929\n- id: '7782928'\n  job: '370874'\n  level: ''\n  log_end_line: '923'\n  log_start_line: '923'\n  logged: '2020-11-19 07:06:12.073952+00:00'\n  measurement: 0E-10\n  metadata: {case: alert, definition: 0_dmesg, measurement: '0', result: pass, units: lines}\n  name: alert\n  result: pass\n  suite: 0_dmesg\n  unit: lines\n  url: /results/testcase/7782928\n- id: '7782927'\n  job: '370874'\n  level: ''\n  log_end_line: '921'\n  log_start_line: '921'\n  logged: '2020-11-19 07:06:12.073920+00:00'\n  measurement: 0E-10\n  metadata: {case: crit, definition: 0_dmesg, measurement: '0', result: pass, units: lines}\n  name: crit\n  result: pass\n  suite: 0_dmesg\n  unit: lines\n  url: /results/testcase/7782927\n", "1_bootrr": "- id: '7782943'\n  job: '370874'\n  level: ''\n  log_end_line: '972'\n  log_start_line: '972'\n  logged: '2020-11-19 07:06:18.378199+00:00'\n  measurement: None\n  metadata: {case: deferred-probe-empty, definition: 1_bootrr, result: pass}\n  name: deferred-probe-empty\n  result: pass\n  suite: 1_bootrr\n  unit: ''\n  url: /results/testcase/7782943\n"}, "status_string": "complete", "end_time": "2020-11-19 07:06:18.385678+00:00"}
```

Comment:
Maybe this could be reformatted with indentation?  Its pretty hard to read that way.